packages/eslint-plugin/tests/rules/no-unsafe-call.test.ts

```diff
@@ -163,5 +233,22 @@ const methods = {
         },
       ],
     },
+    {
+      code: `
+let value: NotKnown;
+value();
+      `,
+      errors: [
+        {
+          messageId: 'unsafeCall',
+          line: 3,
+          column: 1,
+          endColumn: 6,
+          data: {
+            type: '`error` type',
```

Comment:
shouldn't be just "error"?

```suggestion
            type: '`error`',
```

Reply:
Looks like I forgot to change it.